server/env.py

```diff
@@ -5,17 +5,20 @@ class Environment(BaseSettings):
     """
     This is (some of) the environment variables used by the app.
 
-    In production, we specify different values for this variables
+    In PRODUCTION, we specify different values for this variables
```

Comment:
Aqui, pode manter como estava mesmo. Pois está falando sobre o ambiente de produção, não a variável `production`.

```suggestion
    In production, we specify different values for this variables
```